<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StormnodeList</class>
 <widget class="QWidget" name="StormnodeList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer0">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">QTabWidget:tab-bar { alignment: left; }
QTabBar { alignment: left; }
</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabMyStormnodes">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <attribute name="title">
         <string>My Stormnodes</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_note">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="updateNote">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Note: Status of your stormnodes in local wallet can potentially be slightly incorrect.&lt;br /&gt;Always wait for wallet to sync additional data and then double check from another node&lt;br /&gt;if your node should be running but you still see &quot;MISSING&quot; in &quot;Status&quot; field.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidgetMyStormnodes">
             <property name="minimumSize">
              <size>
               <width>695</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color:(255,255,255);</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Alias</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Address</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Protocol</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Active</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Last Seen</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Pubkey</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="startAllButton">
               <property name="styleSheet">
                <string notr="true">QWidget#StormnodeList .QPushButton#startAllButton {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#startAllButton:hover {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #BA77D4, stop: .1 #9F2DCC, stop: .95 #9F2DCC, stop: 1 #7400A1);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#startAllButton:pressed {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}</string>
               </property>
               <property name="text">
                <string>Start &amp;all</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startButton">
               <property name="styleSheet">
                <string notr="true">QWidget#StormnodeList .QPushButton#startButton {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#startButton:hover {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #BA77D4, stop: .1 #9F2DCC, stop: .95 #9F2DCC, stop: 1 #7400A1);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#startButton:pressed {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}</string>
               </property>
               <property name="text">
                <string>S&amp;tart alias</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startMissingButton">
               <property name="styleSheet">
                <string notr="true">QWidget#StormnodeList .QPushButton#startMissingButton {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#startMissingButton:hover {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #BA77D4, stop: .1 #9F2DCC, stop: .95 #9F2DCC, stop: 1 #7400A1);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#startMissingButton:pressed {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}</string>
               </property>
               <property name="text">
                <string>Start &amp;MISSING</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="UpdateButton">
               <property name="styleSheet">
                <string notr="true">QWidget#StormnodeList .QPushButton#UpdateButton {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#UpdateButton:hover {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #BA77D4, stop: .1 #9F2DCC, stop: .95 #9F2DCC, stop: 1 #7400A1);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}

QWidget#StormnodeList .QPushButton#UpdateButton:pressed {
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: .01 #8F5BA3, stop: .1 #771F99, stop: .95 #771F99, stop: 1 #520072);
border:0;
border-radius:3px;
color:#ffffff;
font-size:12px;
font-weight:bold;
padding-left:25px;
padding-right:25px;
padding-top:5px;
padding-bottom:5px;
}</string>
               </property>
               <property name="text">
                <string>&amp;Update status</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="autoupdate_label">
               <property name="text">
                <string>Status will be updated automatically in (sec):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="secondsLabel">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabAllStormnodes">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <attribute name="title">
         <string>All Stormnodes</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidgetStormnodes">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color:(255,255,255);</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Address</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Protocol</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Status</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Last Seen</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pubkey</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_filter">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
</string>
             </property>
             <property name="text">
              <string>Filter List:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterLineEdit">
             <property name="toolTip">
              <string>Filter stormnode list</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_count">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>Node Count:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="countLabel">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
